ingest/router.go

```diff
@@ -195,7 +196,7 @@ type BatchPayload struct {
 func (r *Router) sendToRotor(c *gin.Context, ingestMessageBytes []byte, stream *StreamWithDestinations, sendResponse bool) (asyncDestinations []string, tagsDestinations []string, rError *appbase.RouterError) {
 	var err error
 	if stream.BackupEnabled {
-		backupTopic := fmt.Sprintf("%sin.id.%s_backup.m.batch.t.backup", r.config.KafkaTopicPrefix, stream.Stream.WorkspaceId)
+		backupTopic := fmt.Sprintf("%sin.id.%s.backup.m.batch.t.backup", r.config.KafkaTopicPrefix, stream.Stream.WorkspaceId)
```

Comment:
I couldn't find anywhere else where this backup topic name is referenced, so I think this change won't break anything elsewhere - please let me know if I missed anything.

Reply:
UPDATE: found some `_backup` strings in the `jitsu` repo:

- https://github.com/jitsucom/jitsu/blob/59a4b13c3f0bf48a25ffef8a94483d7eeb50afde/webapps/ee-api/pages/api/s3-connections.ts#L34
- https://github.com/jitsucom/jitsu/blob/59a4b13c3f0bf48a25ffef8a94483d7eeb50afde/webapps/ee-api/scripts/sql/02-backup-connections.sql#L3

Are these related?